lib/yabeda/dsl/class_methods.rb

```diff
@@ -58,8 +58,12 @@ def histogram(*args, **kwargs, &block)
       #
       # @param name [Symbol] Name of default tag
       # @param value [String] Value of default tag
-      def default_tag(name, value)
-        ::Yabeda.default_tags[name] = value
+      def default_tag(name, value, group: nil)
+        if group
+          Yabeda::Tags.tag_group(group)[name] = value
+        else
+          Yabeda.default_tags[name] = value
```

Comment:
I considered updating `Yabeda.default_tags` itself to be a hash of hashes and let nil be the key that accessed top level tags.